.github/workflows/ci.yml

```
@@ -0,0 +1,275 @@
+        path: vendor
+        key: composer-${{ runner.os }}-8.4-${{ hashFiles('composer.lock') }}
+        restore-keys: composer-${{ runner.os }}-8.4-
+
+    - name: Install Dependencies
+      run: composer install --no-interaction --prefer-dist --optimize-autoloader --dev
+
+    - name: Run Pint (Code Style)
+      run: ./vendor/bin/pint --test
+
+    - name: Run Duster (Additional Checks)
+      run: ./vendor/bin/duster lint
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)